packages/fast-components-react-msft/src/call-to-action/call-to-action.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import { Foundation, HandledProps } from "@microsoft/fast-components-react-base";
+import { Hypertext } from "@microsoft/fast-components-react-base";
+import { CallToActionAppearance, ICallToActionHandledProps } from "./call-to-action.props";
+import { IMSFTCallToActionClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import { get } from "lodash-es";
+
+// tslint:disable-next-line:max-line-length
+class CallToAction extends Foundation<ICallToActionHandledProps & IManagedClasses<IMSFTCallToActionClassNameContract>,  React.AllHTMLAttributes<HTMLElement>, {}> {
+    public static defaultProps: Partial<ICallToActionHandledProps> = {
+        appearance: CallToActionAppearance.primary
+    };
+
+    protected handledProps: HandledProps<ICallToActionHandledProps & IManagedClasses<IMSFTCallToActionClassNameContract>> = {
+        appearance: void 0,
+        href: void 0,
+        managedClasses: void 0
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): JSX.Element {
+        return (
+            <Hypertext
```

Comment:
Can you elaborate on the reason you're returning Hypertext here? CTA could be a button...we've definitely seen that before and I believe it's a big reason that we created the button component in the way we did. It seems like we could reduce complexity and code simply by using the MSFT button. There is way more alignment with `<Button />` than there is with `<Hypertext />`.

Reply:
This is a good point, might suggest raising this in today's standup.

Reply:
Seems like a disconnect. I had a conversation with @janechu and @nicholasrice and this was the conclusion. To extend hypertext in MSFT only, as CTA has always been an anchor (based off Fluent). We do not have a glyph system to use yet, so these are base64 encoded images. The animations add classNames and JSS. Semantically speaking, having a CTA component for usage in Microsoft ecosystems makes a lot more sense to me than strong-arming the button to behave differently when a glyph is added.

Reply:
You can have a CTA in an app that needs to be a button. From my viewpoint this is undoubtedly built out of the base button due to functional requirements of being either an anchor or a button. This seems like carry-over from previous implementations and I think there is a lot of reuse between button and CTA in MSFT styles. There is very little delta between these two. The difference is primarily in style enums, which is carryover from a previous implementation. As an aside - I would expect justified to exist here as well. Perhaps CTA is an implementation or extension of Button if need be, or we build it all new with the duplication of Button - at this point I definitely do not see this as Hypertext though.